Scenes/Shared/EventSystem.gd

```diff
@@ -187,15 +187,15 @@ func moneyDecrease():
 func disableMapRegion():
 	var region = randi() % 17
 	var random_adder = randi() % 20
-	get_node("/root/Session/MapUI").random_region_event(region, true, random_adder)
+	var region_name = get_node("/root/Session/MapUI").random_region_event(region, true, random_adder)
 	
-	prompt("region disabled: %s" % region)
+	prompt("%s is disabled" % region_name)
```

Comment:
Chinange ko rin yung text, so ulitin ko na lang to mamaya pag na merge na

Reply:
Siguro wag mo na munang ichange rin yung event system, kasi binago yung text. Iedit ko na lang siya to take advantage of random_adder

Reply:
Rinesolve ko na lang yung conflict dito